clients/admin-ui/src/features/common/Stepper.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Stack, Text } from "@fidesui/react";
+import React from "react";
+
+import steps from "../config-wizard/constants";
+import { StepperCircleIcon, VerticalLineIcon } from "./Icon";
+
+interface Props {
+  activeStep: number | null;
+}
+
+const Stepper = ({ activeStep }: Props) => (
```

Comment:
nit: great idea to make this into a common component! but since it's common, I think the stepper should take a list of steps as props, as opposed to always importing the config wizard constants. so that if we do want to reuse this Stepper in the future, we can pass in different steps.